app/javascript/App/bundles/common/utils/images.ts

```diff
@@ -0,0 +1,91 @@
+export const bellIcon = `data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="14" height="17" viewBox="0 0 14 17">
```

Comment:
I think it may be better to keep these as separate `.svg` files than merged into a single `.ts`. I could see it easily getting out of hand. It's probably not too bad to keep them as `require(...)` calls now until we configure webpack to accept importing `.svg`s

Reply:
I could separate them to a file a piece rather than all in one? Not sure if it's still an issue with server side rendering implemented, but before the images would sometimes pop in since they were referencing an actual image rather than inline.

Reply:
I think there is a webpack plugin/loader for us to do that for all `.svg`s automatically. Maybe beyond the scope of this PR though so I would keep these as `.svg`s and deal with it later. What do you think @hanchang?

Reply:
Yes, def beyond scope of this PR/ticket, let's keep as is for now and @anthotsang please spin up a new ticket to handle this in the Platform board under the Inbox row and we'll prioritize for another sprint, thanks!

Reply:
ok restored to .svg files and created ticket for the future.